pkg/processor/trigger/cron/trigger.go

```diff
@@ -80,7 +81,17 @@ func newTrigger(logger logger.Logger,
 	} else if configuration.Schedule != "" {
 		newTrigger.tickMethod = tickMethodSchedule
 
-		newTrigger.schedule, err = cronlib.Parse(configuration.Schedule)
+		// make sure the given schedule structure contains all 6 fields (Second, Minute, Hour, ...)
+		splitSchedule := strings.Split(configuration.Schedule, " ")
+		if len(splitSchedule) != 6 {
+			return nil, errors.Errorf("Cron trigger schedule doesn't have all needed 6 fields: %+v", configuration.Schedule)
+		}
+
+		// change the seconds to be automatically 0 (to prevent the method from running every second)
```

Comment:
I believe the comment should be `Cron schedule kind does not support seconds`

in order to support seconds-base interval, user can use Cron type of interval